io.github.hrkfdn.ncspot.appdata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019-2022 Henrik Friedrichsen -->
+<component type="desktop">
+  <id>io.github.hrkfdn.ncspot</id>
+  <launchable type="desktop-id">io.github.hrkfdn.ncspot.desktop</launchable>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+  <name>ncspot</name>
+  <summary>Cross-platform ncurses Spotify client written in Rust</summary>
```

Comment:
```suggestion
  <summary>Cross-platform ncurses Spotify client</summary>
```

This doesn't matter to users.

Reply:
it actually has meaning for the end user as it qualifies the UI. Albeit "text based" or "terminal based" is probably more descriptive.